README.md

```diff
@@ -14,7 +14,7 @@ You can provide a postman collection and environment to be tested in one of two
 1. Provided in your github repo
     ```hcl
     module "postman_test_lambda" {
-      source = "github.com/byu-oit/terraform-aws-postman-test-lambda?ref=v2.3.1"
+      source = "github.com/byu-oit/terraform-aws-postman-test-lambda?ref=v2.4.1"
```

Comment:
why not just `2.4.0`?

Reply:
Because sometimes I don't use my brain. 😂 Should be fixed now